README.md

```diff
@@ -30,6 +30,7 @@ There are two ways to configure your plugin to submit data to Datadog:
 
 * **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
   - Build Logs collection only works with a full Datadog Agent installed.
+  - For metrics submitted from an external host, the Datadog Agent requires the following configuration for DogStatsD: `dogstatsd_non_local_traffic: true`. This can be configured using the `datadog.yaml` configuration file. 
```

Comment:
Can you add a link to `configuration file` leading here: `https://docs.datadoghq.com/developers/dogstatsd/?tab=containeragent#` like so:

```suggestion
  - For metrics submitted from an external host, the Datadog Agent requires the following configuration for DogStatsD: `dogstatsd_non_local_traffic: true`. This can be configured using the `datadog.yaml` [configuration file][17]. 
```

Then add to bottom:

```
[17]: https://docs.datadoghq.com/developers/dogstatsd/?tab=containeragent#
```

Reply:
Thanks for the note! I should have just applied the recommended changes.